.github/workflows/sauce.yml

```diff
@@ -0,0 +1,27 @@
+name: Sauce Labs
+on: push
+This conversation was marked as resolved by marcuslyons
+ Show conversation
```

Comment:
Copy-paste mistake?

Reply:
Yeah, sorry about that.

Removed in 96172d1